### ISSUE TITLE
Returning func be able to handle nil

### ISSUE THREAD

**Issue by randallmlough** [doug-martin/goqu#140]
**Is your feature request related to a problem? Please describe.**
It would be nice if `Returning` function could be nil and if so act is if it's not there. 

I use your (_awesome_) library as a helper func to build queries across my application. In some cases I want to return values like an id or the whole updated row, but then sometimes I don't need to. It doesn't make a lot of sense to create additional logic just for a `Returning` stmt.

**Note**: The desired behavior works if the `Returning` func isn't passed any values `.Returning()` but if you pass it the value of `nil` it panics (because of reflect). As a result since this returning clause is in a helper func it will always be passed some value, even if it's just nil.

**Describe the solution you'd like**
Maybe a check before reflect is run?
```go
func (ud *UpdateDataset) Returning(returning ...interface{}) *UpdateDataset {
	if isNil(returning){
              return ud
        }
	// continue...
}
func isNil(v interface{}) bool {
	switch t := v.(type) {
	case []interface{}:
		if len(t) <= 0 || (len(t) == 1 && t[0] == nil){
			return true
		}
	default:
		if t == nil {
			return true
		}
	}
	return false
}
```

**Comment by doug-martin** [comment]
This should be fixed by #142. Ill update and close once published.

**Comment by doug-martin** [comment]
Just published! Let me know if you have any more issues.
